### DISCUSSION TITLE
Configurate doctrine_migration_versions table in custom schema instead of public

### DISCUSSION THREAD

**Discussion by wardaari** [symfony/symfony#45059] [Q&A] [answered]
Is there a way for the automatic `doctrine_migration_versions`to be stored in a schema other than `public`?

Right now I have a schema called `inschrijving` and all the entities properties are stored as tables and columns within that schema, but not the doctrine migration versions table.

Here's how my entities are stored:
[![enter image description here][1]][1]

And the `doctrine_migration_versions` is stored in the `public` schema:
[![enter image description here][2]][2]

  [1]: https://i.stack.imgur.com/mjx68.png
  [2]: https://i.stack.imgur.com/T02N8.png

I cannot find anywhere how to configure doctrine in order for the **doctrine_migration_versions** to be stored within the **inschrijving** schema instead of the **public** schema

**Comment by stof** [comment, marked as answer]
should be asked in https://github.com/doctrine/DoctrineMigrationsBundle/ or https://github.com/doctrine/migrations, as this is not a question about features provided by Symfony. The Doctrine maintainers are more likely to be able to answer here.

Based on a pure guess (and without trying it), maybe configuring the table name as `inschrijving.doctrine_migration_versions` in the bundle configuration would work (be careful that renaming the table storing the state of applied migrations won't preserve the existing state in the old table, as doctrine/migrations won't know about it anymore)
